modelforge/dataset/dataset.py

```diff
@@ -139,6 +144,7 @@ class NNPInput:
     total_charge: torch.Tensor
     pair_list: Optional[torch.Tensor] = None
     partial_charge: Optional[torch.Tensor] = None
+    box_vectors: Optional[torch.Tensor] = torch.tensor([])
```

Comment:
It would be good to add that into the dataset setup.  I think we still need to think a bit about our "external" interface.  For openmm interface, it seems like if the system is not periodic, box vectors just aren't passed to the function call, so I guess we would know based on that what flags to set.

Reply:
I refactored a bit after putting "is_periodic" into NNPInput.  periodicity is no longer passed to the constructor of the displacement function, but passed every time we call it (just like box vectors).  This should be more portable for the model.

Reply:
that's great!